public/app/features/alerting/unified/rule-list/FilterView.tsx

```diff
@@ -0,0 +1,155 @@
+import { take, tap, withAbort } from 'ix/asynciterable/operators';
+import { useEffect, useRef, useState, useTransition } from 'react';
+import Skeleton from 'react-loading-skeleton';
```

Comment:
There are some skeletons already in the grafana UI library AFAIK. Could we use those, or add a TODO to make sure we update those components to let us reuse them in the future?

Reply:
Looks like there is nothing in `grafana/ui` we could use. The predefined skeletons works for predefined components (like Tag or Badge)